src/csharp/Microsoft.Spark/JvmException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Spark
+{
+    public class JvmException : Exception
```

Comment:
Since this will be a public class, can you add appropriate comments? (e.g. https://github.com/dotnet/spark/blob/ce2317774c25550ea3784a1af4f02befd2eea70b/src/csharp/Microsoft.Spark/Sql/DataFrame.cs#L35-L40)

Reply:
Hopefully this is OK.
```cs
    /// <summary>
    /// Contains the message returned from the Jvm Bridge on an error
    /// </summary>
```